src/components/docs/Docs/Sidebar/Sidebar.module.scss

```diff
@@ -1,80 +1,175 @@
+        }
+    }
+}
+
+.menuHeader {
+    padding: 20px 0 20px 0;
     font-size: 16px;
-    font-weight: bold;
-    line-height: 1.88;
-    opacity: 1;
+    line-height: 24px;
+    font-weight: 700;
+    color: #333333;
+    border-top: 2px solid var(--gray-01);
+}
+
+.menuItem {
+    width: 236px;
+    height: 24px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+
+    .selected {
+        background-color: var(--gray-01);
+        font-size: 16px;
+        font-weight: bold;
+        line-height: 1.88;
+        opacity: 1;
 
-    &::before {
-      visibility: visible;
-      height: 100%;
+        &::before {
+            visibility: visible;
+            height: 100%;
+        }
     }
-  }
 }
 
 .sidebartitle {
-  margin: 0 0 16px 2px;
-  font-size: 32px;
-  line-height: 1.25;
-  color: #333333;
-  font-weight: bold;
+    margin: 18px 0 16px 2px;
+    font-size: 32px;
+    line-height: 1.25;
+    color: #333333;
+    font-weight: bold;
+    border-bottom: 2px solid var(--gray-01);
+}
+
+.dropdown {
+    width: 236px;
+    min-height: 24px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+
+    .arrow {
+        background-size: 22px;
+        background-position: center;
+        background-repeat: no-repeat;
+        cursor: pointer;
+        background-image: url(/img/arrow_up.svg);
+        transform: rotate(-180deg);
+        transition: transform 0.2s;
+        width: 22px;
+        height: 22px;
+        &.down {
+            transform: rotate(0deg);
+        }
+    }
+
+    .dropdownBtn {
+        position: relative;
+        opacity: 0.72;
+        margin-bottom: 14px;
+        line-height: 1.88;
+        color: #333333;
+        padding: 0 24px 0 10px;
+        display: flex;
+        align-items: center;
+        cursor: pointer;
+        justify-content: space-between;
+    }
+    .boldText {
+        font-weight: 700;
+        color: #000;
+    }
+    .dropdownList {
+        max-height: 700px;
+        overflow: auto;
+        .dropdownContent {
+            width: 236px;
+            height: 24px;
+            font-weight: 400;
+            font-size: 16px;
+            padding: 10px 24px 10px 17px;
+            line-height: 24px;
+            margin-top: 14px;
+            color: #333333;
+        }
+        .selected {
+            background-color: var(--gray-01);
+            font-size: 16px;
+            font-weight: bold;
+            line-height: 1.88;
+            opacity: 1;
+
+            &::before {
+                visibility: visible;
+                height: 100%;
+            }
+        }
+    }
 }
 
-[data-state*="responsive.mobile"] {
-  .sidebarleft {
-    display: none;
-  }
+[data-state*='responsive.mobile'] {
+    .sidebarleft {
+        display: none;
+    }
 }
 
-[data-state~="responsive.desktopLaptopM"] {
-  .sidebarleft {
-    padding: 0;
-  }
+[data-state~='responsive.desktopLaptopM'] {
+    .sidebarleft {
+        padding: 0;
+    }
```

Comment:
at @sanjam-deriv I believe this is css of your doing. Hope that nesting in css can be maintainable for others